### PR TITLE
docs: unescape html entities

### PR DESCRIPTION
**Pull request by JeanMeche**
adev supports regular characters in place of entity. Let's use them to make the ease the work for the editors.

### CODE REVIEW

File: adev/src/content/best-practices/a11y.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ The following example shows how to find and focus the main content header in the
 
 <docs-code language="typescript">
 
-router.events.pipe(filter(e =&gt; e instanceof NavigationEnd)).subscribe(() =&gt; {
+router.events.pipe(filter(e => e instanceof NavigationEnd)).subscribe(() => {
   const mainHeader = document.querySelector('&num;main-content-header')
```
Comment (sonukapoor) [inline adev/src/content/best-practices/a11y.md:107]:
```suggestion
  const mainHeader = document.querySelector('#main-content-header')
```

File: adev/src/content/best-practices/style-guide.md [hunk 2]
```diff
@@ -151,14 +151,14 @@ Consistent conventions make it easy to quickly identify and reference assets of
...
-| <docs-code hideCopy language="typescript"> &commat;Component({ &hellip; }) &NewLine;export class HeroDetailComponent { } </docs-code>                      | hero-detail.component.ts |
-| <docs-code hideCopy language="typescript"> &commat;Directive({ &hellip; }) &NewLine;export class ValidationDirective { } </docs-code>                      | validation.directive.ts |
-| <docs-code hideCopy language="typescript"> &commat;NgModule({ &hellip; }) &NewLine;export class AppModule </docs-code>                                     | app.module.ts |
-| <docs-code hideCopy language="typescript"> &commat;Pipe({ name: 'initCaps' }) &NewLine;export class InitCapsPipe implements PipeTransform { } </docs-code> | init-caps.pipe.ts |
-| <docs-code hideCopy language="typescript"> &commat;Injectable() &NewLine;export class UserProfileService { } </docs-code>                                  | user-profile.service.ts |
+| <docs-code hideCopy language="typescript"> @Component({ … }) &NewLine;export class AppComponent { } </docs-code>                             | app.component.ts |
```
Comment (sonukapoor) [inline adev/src/content/best-practices/style-guide.md:154]: Can we wrap these into the new line and get rid of `&NewLine;`?

**Review by sonukapoor — APPROVED** [review]
Thanks for making these changes.

File: adev/src/content/best-practices/a11y.md (hunk 1 above)
Comment (JeanMeche) [reply adev/src/content/best-practices/a11y.md:107]: Good catch

File: adev/src/content/best-practices/style-guide.md (hunk 2 above)
Comment (JeanMeche) [reply adev/src/content/best-practices/style-guide.md]: I've replaced it with `<br>` as we can just have a new line in a table.

**Review by sonukapoor — APPROVED** [review]
LGTM.

**Review by bencodezen — APPROVED** [review]
Agreed! This is a relief to move away from the escaped characters. Thanks for the PR!

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit b2ba94543637a5e94af4411bae62891c43ebe359.

**Pull request closed without merging**